libs/datasets/taglib.py

```diff
@@ -328,6 +329,22 @@ def content(self) -> str:
         return json.dumps(d, separators=(",", ":"))
 
 
+@dataclass(frozen=True)
+class DropFutureObservation(TagInTimeseries):
+    TAG_TYPE = TagType.DROP_FUTURE_OBSERVATION
+    after: datetime.date
+
+    @classmethod
+    def make_instance(cls, *, content: str) -> "TagInTimeseries":
+        content_parsed = json.loads(content)
+        return cls(after=datetime.date.fromisoformat(content_parsed["after"]),)
```

Comment:
Is that trailing `,` intentional / necessary?

Reply:
Nope. Upgrading black fixes this but also makes lots of other changes so won't do it in this PR.